app/scripts/controllers/transactions/index.test.js

```diff
@@ -331,7 +331,7 @@ describe('Transaction Controller', function () {
     });
   });
 
-  describe('#addTransaction', function () {
+  describe('_addTransaction', function () {
```

Comment:
This one is referring to the public method can keep the `#`.

There is also a private `_addTransaction` but that's not tested directly.

Reply:
Good catch, thanks